cdap-ui/app/directives/log-viewer/log-viewer.html

```diff
@@ -112,55 +112,33 @@
              infinite-scroll-disabled="LogViewer.viewLimit >= LogViewer.data.length"
              infinite-scroll-container="'.logs-table'"
              >
-        <tr ng-repeat="entry in LogViewer.data | filter:LogViewer.filterByStartDate | filter:searchText | filter:LogViewer.eventFilter | limitTo: LogViewer.viewLimit track by $index">
-          <!-- TIME -->
-          <td class="time"
-              ng-if="LogViewer.configOptions.time && !LogViewer.hiddenColumns.time">
-              {{ entry.log.displayTime }}
+
+        <tr ng-class="{'selected-row' : entry.selected, 'has-stack-trace' : entry.log.stackTrace.length > 0, 'row-selected' : $index+1  < LogViewer.data.length && LogViewer.data[$index + 1].stackTrace }" ng-repeat="entry in LogViewer.data | filter:LogViewer.filterByStartDate | filter:searchText | filter:LogViewer.eventFilter | limitTo: LogViewer.viewLimit track by $index" ng-click="LogViewer.showStackTrace($index)">
```

Comment:
What is the difference between `selected-row` and `row-selected`? Its little hard to understand what they are representing.